EMEATT-XXX: fixes for the latest flex form config.

prosidney opened the pull request.

Commit:
EMEATT-XXX: fixes for the latest flex form config.

Commit:
EMEATT-XXX: incremented the version.

TimDG commented on flex.js:
```diff
@@ -1,7 +1,7 @@
 // @name         Flex usability
 // @namespace    http://tampermonkey.net/
-// @version      0.3.0
+// @version      0.3
```
You need to up the version to have the script change picked up

TimDG commented on flex.js:
```diff
@@ -145,7 +145,7 @@
             //Sunday
             $shiftNameSelect.val("OT 150%");
         } else {
-            $shiftNameSelect.val("Day Shift");
```
What about the other dates (Saturday/Sunday)?

TimDG committed:
Update the shift name in both locations

TimDG committed:
Added a confirmation dialog.

Hopefully this should prompt people to check that the values that were automatically entered are actually correct.
